pygfx/utils/text/_atlas.py

```diff
@@ -211,15 +226,24 @@ def _set_new_glyphs_array(self, size):
         """
         assert size > 0
 
+        # Set the fill value to 255 to debug the text. it should create
+        # slight artifacts around the text, but it should be very hard to see.
+        # However our tests should be able to see it
+        fill_value = 255
```

Comment:
actually, if you watch the waterfall demo with the with this, you can see a faint "grey" or light blue color appear around some of the characters. it is most prominent around characters in the graveyard.
![Screenshot from 2025-02-26 18-06-50](https://github.com/user-attachments/assets/3fd23e94-a822-4d9d-b9d0-27c6c2ba842a)

![image](https://github.com/user-attachments/assets/da4c0679-738f-42c9-9276-53116b868e45)
![image](https://github.com/user-attachments/assets/707b7fb4-fb42-4727-b423-6e7353a38bfe)
![image](https://github.com/user-attachments/assets/7312775e-c09a-43f6-90d3-942f8e02a417)